en/guide/mavlink_version.md

```diff
@@ -1,27 +1,52 @@
 # MAVLink Versions
 
-MAVLink is deployed in two major versions: *MAVLink 1.0* and [MAVLink 2.0](../guide/mavlink_2.md).
+MAVLink has been deployed in a number of versions:
+- [MAVLink 2.0](../guide/mavlink_2.md): Current/recommended major version. Adopted by major users early 2017. 
+- *MAVLink v1.0*: Widely adopted around 2013. Still used by a number of legacy peripherals.
+
+The *MAVLink 2.0* C/C++ and Python libraries are backwards compatible with MAVLink 1.0 (support both protocols).
+[Version Handshaking](#version_handshaking) and [Negotiating Versions](#negotiating_versions) explain how to choose which version is used.
+  
+> **Note** *MAVLink v0.9* is a pre-release version that is no longer supported. 
+  The associated message set deleted in August 2018. 
+  Legacy code may be present in generator and test code. 
+
+
+## Determining Protocol/Message Version
+
+A library's MAVLink support can be determined in a number of ways:
+* [AUTOPILOT_VERSION](../messages/common.md#AUTOPILOT_VERSION)`.capabilities` can be checked against the [MAV_PROTOCOL_CAPABILITY_MAVLINK2](../messages/common.md#MAV_PROTOCOL_CAPABILITY_MAVLINK2) flag to verify MAVLink 2 support.
+* [PROTOCOL_VERSION](../messages/common.md#PROTOCOL_VERSION).`version` contains the MAVLink version number multiplied by 100: v1.0 is 100, <!-- v2.0 is 200, --> v2.3 is 203 etc.
```

Comment:
1. Confirm that the intention is that the "00" is for minor version.
1. That implies the max number of minor revisions we can have is 99, even if we store this elsewhere as 256.

Reply:
I'm on the fence on this. People were completely confused about it not being 2.x originally.